src/app/core/services/advance-request.service.ts

```
@@ -508,37 +523,130 @@ export class AdvanceRequestService {
+      areq_approved_at: advanceRequestPlatform.last_approved_at,
+      areq_created_at: advanceRequestPlatform.created_at,
+      areq_currency: advanceRequestPlatform.currency,
+      areq_id: advanceRequestPlatform.id,
+      areq_notes: advanceRequestPlatform.notes,
+      areq_org_user_id: advanceRequestPlatform.employee_id,
+      areq_project_id: advanceRequestPlatform.project_id,
+      areq_purpose: advanceRequestPlatform.purpose,
+      areq_source: advanceRequestPlatform.source,
+      areq_state: this.getAdvanceRequestState(advanceRequestPlatform.state),
+      areq_updated_at: advanceRequestPlatform.updated_at,
+      ou_department: advanceRequestPlatform.employee.department && advanceRequestPlatform.employee.department.name,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)